stan/math/prim/meta/vector_seq_view.hpp

```diff
@@ -29,45 +30,17 @@ class vector_seq_view {};
  * only allows std::vectors as the container type, since we would have
```

Comment:
Don't think this is true anymore.

Reply:
Actually it is. It is just written in a confusing way. This is meant as the container `vector_seq_view` treats as vector and iterates over. Eigen types are treated kinda as kinda scalars by `vector_seq_view`. Updated to hopefuly make it a bit less confusing.

Reply:
Oh yeah I see. The docs are written for like the high level container and just sorta copied in multiple places.